tests/xrdp/gfx/gfx.toml

```diff
@@ -0,0 +1,40 @@
+[codec]
+order = [ "H264", "RFX" ]
```

Comment:
Since you're checking for H264 or H.264 in your string compare, how about making this one "H.264" rather than "H264"?

Reply:
Agreed.